Tests/AutoCollection/Statsbeat.tests.ts

```diff
@@ -171,17 +171,17 @@ describe("AutoCollection/Statsbeat", () => {
         it("Track counts", (done) => {
             statsBeat.enable(true);
             const sendStub = sandbox.stub(statsBeat, "_sendStatsbeats");
-            statsBeat.countRequest(0, "test", 1, true);
-            statsBeat.countRequest(0, "test", 1, true);
-            statsBeat.countRequest(0, "test", 1, true);
-            statsBeat.countRequest(0, "test", 1, true);
-            statsBeat.countRequest(0, "test", 1, false);
-            statsBeat.countRequest(0, "test", 1, false);
-            statsBeat.countRequest(0, "test", 1, false);
-            statsBeat.countRetry(0, "test");
-            statsBeat.countRetry(0, "test");
-            statsBeat.countThrottle(0, "test");
-            statsBeat.countException(0, "test");
+            statsBeat.countRequest(0, "test", 1, true, 200);
+            statsBeat.countRequest(0, "test", 1, true, 200);
+            statsBeat.countRequest(0, "test", 1, true, 200);
+            statsBeat.countRequest(0, "test", 1, true, 200);
+            statsBeat.countRequest(0, "test", 1, false, 200);
+            statsBeat.countRequest(0, "test", 1, false, 200);
+            statsBeat.countRequest(0, "test", 1, false, 200);
+            statsBeat.countRetry(0, "test", 206);
+            statsBeat.countRetry(0, "test", 206);
+            statsBeat.countThrottle(0, "test", 206);
+            statsBeat.countException(0, "test", "Statsbeat Exception");
```

Comment:
Add assert checking this exception type

Reply:
Added tests to determine the exceptionType field is populated correctly on the metric.